### PR TITLE
LPS-183797 | master

### PR DESCRIPTION
**Pull request by andreefarias**
Ticket: [LPS-183797](https://issues.liferay.com/browse/LPS-183797)

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path [hunk 1]
```diff
@@ -620,6 +620,11 @@
 		<td>//div[contains(@class,'card-body')]//button[contains(text(), '${key_optionDefaulValue}')]</td>
 		<td></td>
 	</tr>
+	<tr>
+		<td>VIEW_ICON_OBJECT_ADMIN</td>
+		<td>//div[(@class='table-list-title')]//*[(@class='lexicon-icon lexicon-icon-view')]</td>
+		<td></td>
+	</tr>
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path:627]: I added a new path to click on `View Icon`.

The View Icon is available when changing the portal language and not adding a new translation to the object/field.

Screenshot

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectTranslations.testcase [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+		task ("Then the custom object cannot be saved without filling the required fields") {
+			Click(locator1 = "ObjectAdmin#VIEW_ICON_OBJECT_ADMIN");
+
+			Button.clickSave();
+
+			AssertElementPresent(
+				locator1 = "Message#WARNING_FEEDBACK",
+				value1 = "Required");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectTranslations.testcase:67]: Right now, we don't actually assert that the user cannot save the object, we only check if the error message appears. Is it possible to do the following?  
- Click on the icon
- Fill out one of the required fields
- Click Save
- Assert the error message appears
- Assert the field wasn't updated

**Review by timpak — COMMENTED** [review]
@andreefarias can you take a look at my question below? If the answer is no, you can forward this. If the answer is yes, can you make the changes and then forward it? I think you need to remove 82114d791d64f09a5f33c91a5df5dd0bc35d2e30 from this branch because it's causing a merge conflict.

**Pull request closed without merging**